Resources/Maps/_LostParadise/LPPKerberos.yml

```
@@ -18696,10 +18696,10 @@ entities:
           - 0
           - 0
         - volume: 2500
-          temperature: 293.15
+          temperature: 235
           moles:
-          - 21.6852
-          - 81.57766
+          - 21.824879
```

yamllint

[error] wrong indentation: expected 12 but found 10

(indentation)